lib/view/account_view.dart

```diff
@@ -157,25 +157,37 @@ Widget accountItem(BuildContext context, Account account,
   );
 }
 
-Future<Pair<String, String>> getAddressBalance(
-    String address, CryptoType cryptoType) async {
-  final tokenDao = injector<TokenDao>();
-  final tokens = await tokenDao.findTokenIDsOwnersOwn([address]);
-  final nftBalance =
-      "${tokens.length} ${tokens.length == 1 ? 'nft'.tr() : 'nfts'.tr()}";
+Future<Pair<String, String>?> getAddressBalance(
+    String address, CryptoType cryptoType,
+    {bool getNFT = true, int minimumCryptoBalance = 0}) async {
+  late String nftBalance;
+  if (getNFT) {
+    final tokenDao = injector<TokenDao>();
+    final tokens = await tokenDao.findTokenIDsOwnersOwn([address]);
+    nftBalance =
+        "${tokens.length} ${tokens.length == 1 ? 'nft'.tr() : 'nfts'.tr()}";
+  } else {
+    nftBalance = '';
+  }
   switch (cryptoType) {
     case CryptoType.ETH:
       final etherAmount = await injector<EthereumService>().getBalance(address);
+      if (etherAmount.getInWei < BigInt.from(minimumCryptoBalance)) {
+        return null;
+      }
       final cryptoBalance =
           '${EthAmountFormatter().format(etherAmount.getInWei)} ETH';
       return Pair(cryptoBalance, nftBalance);
```

Comment:
the cryptoBalance and nftBalance are not related to each other. So we can at first get cryptoBalance (can check minimumCryptoBalance after this step), then get nftBalance and return Pair

Reply:
resolved